pkg/process/config/config.go

```diff
@@ -207,6 +209,7 @@ func NewDefaultAgentConfig(canAccessContainers bool) *AgentConfig {
 		ClosedChannelSize:     500,
 		ConntrackMaxStateSize: defaultMaxTrackedConnections * 2,
 		ConntrackRateLimit:    500,
+		OffsetGuessThreshold:  400,
```

Comment:
I know this is the case for other settings that were already present as well - so I'm not entirely sure this is the right PR to address this, but probably something you need to keep in mind for the future - we're repeating this `400` number all over the place, it should be a `const` in one place, and should be referenced everywhere else (kind of like what you've done with the `maxOffsetThreshold`). If you can fix it now cleanly, and set an example on how this should be done for all similar setting in the sysprobe, it would be great.

Reply:
Hey, apologies, but I don't think I can address this in this PR. I would need to export it to access it in another struct in the ebpf package, which I don't want to arbitrarily do. 

Since this is largely for support, I don't want to add the extra complexity and potentially miss getting this into 7.21 (if that's okay).